src/Domain/DomainHelpers.hpp

```diff
@@ -162,6 +162,9 @@ size_t which_wedge_index(const ShellWedges& which_wedges);
  * \param radial_distribution Select the radial distribution of grid points in
  * the spherical shells.
  * \param which_wedges Select a subset of wedges.
+ * \param opening_angle sets the opening angle of the two half wedges that
```

Comment:
Add a sentence that this only has an effect if `use_half_wedges` is turned on. Also maybe name something like `half_wedges_opening_angle` to clarify that this number applies to the half wedges? Default to `M_PI_4` in that case (and multiply by 2 to get the opening angle of the full wedge). This is an attempt to clarify what "opening angle" means for a full envelope of wedges, other suggestions welcome.

Reply:
I want to keep `opening_angle` as the angle for the full wedge (or both halves combined) because this makes it easier to compare with the endcap full wedge which is also given an `opening_angle`, but I'll had that this parameter has no effect if `use_half_wedges` is turned off

Reply:
Ok